frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1032,7 +1033,7 @@ function ReaderRolling:showEngineProgress(percent)
         local x = 0
         local y = 3
```

Comment:
`local y = Size.margin.small`  (`Screen:scaleBySize(2)`, which hopefully makes 3 on your device :)

Reply:
done

Reply:
Not likely. :-P The old Kindle 800x600 @ 167 is no longer our baseline since we switched it to an easier 160, but that extra 4 % will only round up for certain fractional values (i.e., the `x.45`–`x.49` range that afaik we don't use).

Reply:
on my KT4 i dont think `display_dpi` is set so `Screen:scaleBySize()` will scale input by x1.3 rounded up so `Screen:scaleBySize(2)` should be 3